src/services/cli-http-client.js

```diff
@@ -125,6 +125,17 @@ class CliRequestClient {
       this.logger.debug(options.params);
     }
 
+    // Deep copy headers since we'll be modifying
+    let customHeaders = JSON.parse(JSON.stringify(options.headers));
+    const standardHeaders = ['User-Agent', 'Accept-Charset', 'Connection', 'Authorization', 'Accept', 'Content-Type'];
+    standardHeaders.forEach(header => {
+      delete customHeaders[header];
```

Comment:
I'm paranoid about something else using this client and using a lower-cased `authorization` header (unlikely, but possible) and it getting logged. Prefer to do case-agnostic filtering.

Reply:
I like this new solution.